msbuild/Xamarin.iOS.Tasks/Tasks/CollectMonotouchReferences.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.Build.Definition;
+using Microsoft.Build.Evaluation;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Collections.Generic;
+using System.Xml;
+using System.Linq;
+
+namespace Xamarin.iOS.Tasks
+{
+	public class CollectMonotouchReferences : Task
+	{
+
+		const string MsBuildNamespace = "http://schemas.microsoft.com/developer/msbuild/2003";
+		const string MonoTouchProjectGuid = "6BC8ED88-2882-458C-8E55-DFD12B67127B";
```

Comment:
I think this is the old Classic project guid, which we're not supporting anymore, so I believe this constant (and any related code/checks) can be removed.